frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx

```diff
@@ -77,7 +77,7 @@ export function AdvertiseMobileReplay({
                     Session Replay for {platform} <LemonTag type="highlight">NEW</LemonTag>
```

Comment:
@ioannisj i think we should make this component generic (not only for session replay) and do the very same for Mobile surveys.
The component should get the product name and params so it can render 'any' product with a 'new' tag and instructions inlcuding its state beta|ga